tutor/src/api/index.coffee

```diff
@@ -196,7 +196,7 @@ startAPI = ->
     pattern: 'enrollment/{id}/approve', method: 'PUT'
     onSuccess: 'onApiRequestComplete', onFail: 'setApiErrors')
 
-  connectModelRead(CourseStudentTasks, 'fetch', onSuccess: 'onLoaded', pattern: 'courses/{courseId}/dashboard')
+  connectModelRead(CourseStudentTasks, 'fetch', onSuccess: 'onLoaded', pattern: 'courses/{course.id}/dashboard')
```

Comment:
how is this url pattern used? is it the same router as here? https://github.com/openstax/tutor-js/pull/2378/files#diff-10c23c10ea6ef74f0e103f771283948eR107

i didn't see where you changed any route params from courseId to course

Reply:
Yeah, that change was a side effect of this: https://github.com/openstax/tutor-js/pull/2378/files#diff-27d249dcdd15872c8db5b5f36317bce4L22

StudentTasks used to just store the courseId, but it now stores the course.